src/api/License/index.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright Linkurious SAS 2012 - 2022
+ *
+ * - Created on 2022-03-16.
+ */
+import {Request} from '../../http/request';
+import {LkErrorKey} from '../../http/response';
+
+import {ISubmitLicenseParams, SubmitLicenseResponse} from './types';
+
+export * from './types';
+
+const {INVALID_LICENSE} = LkErrorKey;
+
+export class LicenseAPI extends Request {
+  /**
+   * Verify the supplied license and save it if it’s valid and there is no saved license yet.
+   */
+  public submitLicense(this: Request<SubmitLicenseResponse>, params: ISubmitLicenseParams) {
```

Comment:
To be consistent with my comments on the server PR, I think naming matter for readability and reusability.
Here we can create 2 aliases that do the same thing
`createLicenseIfMissing`
`verifyLicense`

open to discuss.

Reply:
I've renamed this method `saveLicenceIfMissing` as discussed together.